README.md

```diff
@@ -2,11 +2,11 @@
 
 Helps your team control infra costs by pointing potential unused 'zombie' projects.
 
-Owners of the Projects receive messages like this one:
+If you use the Slack integration, the Owners of the Projects receive messages like this one:"
```

Comment:
Remove the `"`

Reply:
Fixed in commit [7984861](https://github.com/kieras/zombie-projects-watcher/pull/63/commits/7984861ad7bbdb6ac95980bfbfe3da0ec31f93b8)